imports/server/guesses.ts

```diff
@@ -57,6 +62,167 @@ function transitionGuess(guess: GuessType, newState: GuessType['state']) {
   }
 }
 
+// It's easier to create one observer per object, rather than try and have a
+// single observer with an $in query. It means we don't have to start/stop the
+// observer. Identical queries from different users will get deduped on the
```

Comment:
> Identical queries from different users will get deduped on the server side

Where does that happen, out of curiosity?  I'm not seeing anything obvious in this code, so I'm wondering if there's some fanciness in Meteor that I'm not aware of.

Reply:
Meteor will dedup identical cursors - happens somewhere around here: https://github.com/meteor/meteor/blob/3cd6e5ebc66216d07e15ef9cc2f326621934e828/packages/mongo/mongo_driver.js#L1295-L1319